easel/lib/services/third_party_services/quick_node.dart

```diff
@@ -0,0 +1,131 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:easel_flutter/env.dart';
+import 'package:easel_flutter/models/storage_response_model.dart';
+
+///* using this because dio is already init with other  base url
+import 'package:http/http.dart' as http;
```

Comment:
As i shared in our chat you can re-use dio by assign the url only when the backend call is needed. Having dio and http in the code base not very pleasing.